src/main/java/com/mkopec/ticketbookingapp/domain/TicketType.java

```diff
@@ -0,0 +1,25 @@
+package com.mkopec.ticketbookingapp.domain;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.math.BigDecimal;
+
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Entity
+@Data
+public class TicketType {
```

Comment:
a dlaczego nie jako stała w kodzie, jakiś enum albo properties?

Reply:
Rozważałem tutaj stworzenie enuma z polem BigDecimal price - wtedy obsługa by była prostsza. Jednak dodanie nowego typu biletu albo zmiana ceny wymaga ingerencji w kodzie, a tak to kończy się na dodaniu lub edycji rekordu w tabeli.

Reply:
super, o to mi chodziło :) generalnie zrobiłeś to świetnie, że jest to właśnie typ w bazie, bo takie rzeczy jak "cena" mogą się częściej zmieniać i nic nie trzeba zmieniać w kodzie przy twoim podejściu